sharedbuffers/mapped_struct.py

```diff
@@ -8056,7 +8056,10 @@ def build(cls, initializer, *p, **kw):
         encode = cls.encode
         def wrapped_initializer():
             for key, value in initializer:
-                yield xxh(encode(key)).intdigest(), value
+                if isinstance(key, (int, long)):
```

Comment:
You need to update the doc string of the method to reflect the fact that it now accepts integers as keys, too.

Reply:
This has docstrings, oh my. Not used to that! haha . will do 👌